dbt_coverage/__init__.py

```diff
@@ -664,10 +687,12 @@ def compute(project_dir: Path = typer.Option('.', help="dbt project directory pa
                                                              "with and fail if current coverage "
                                                              "is lower. Normally used to prevent "
                                                              "coverage drop between subsequent "
-                                                             "tests.")):
+                                                             "tests."),
+            cov_format: Path = typer.Option('string', help="The output format to print, either "
```

Comment:
@vvvito just as I mentioned above, I think it would be great to use the [`Enum`](https://typer.tiangolo.com/tutorial/parameter-types/enum/) here instead -- it would ensure the choices are then nicely documented by themselves 🙂